colibre/scripts/HI_mass_evolution.py

```diff
@@ -68,49 +69,59 @@
 # Observational data plotting
 
 zgrid = np.linspace(0, 4, 50)
+agrid = pow(1 + zgrid, -1)
 
 P20_data = np.genfromtxt(
     f"{arguments.config.config_directory}/{arguments.config.observational_data_directory}"
     "/data/CosmicHIAbundance/raw/Peroux2020_OmegaHI.txt",
     usecols=[1, 2],
 )
-P20eq13 = lambda z, a, b: a * rho_crit0 * (1 + z) ** b
+P20eq13 = lambda z, a, b: a * rho_crit0 * (1.0 + z) ** b
 P20_rhoHI = P20eq13(zgrid, *P20_data[0])
 P20_rhoHI_lo = P20eq13(zgrid, *P20_data[1])
 P20_rhoHI_hi = P20eq13(zgrid, *P20_data[2])
+
+# Cosmology correction (Schaye 2001)
+P20_rhoHI *= comso.h ** -1 * sqrt(cosmo.Om0 * (1.0 + zgrid) ** 3 + comso.Ode0)
+P20_rhoHI /= 0.7 ** -1 * sqrt(0.3 * (1.0 + zgrid) ** 3 + 0.7)
+P20_rhoHI_lo *= comso.h ** -1 * sqrt(cosmo.Om0 * (1.0 + zgrid) ** 3 + comso.Ode0)
+P20_rhoHI_lo /= 0.7 ** -1 * sqrt(0.3 * (1.0 + zgrid) ** 3 + 0.7)
+P20_rhoHI_hi *= comso.h ** -1 * sqrt(cosmo.Om0 * (1.0 + zgrid) ** 3 + comso.Ode0)
+P20_rhoHI_hi /= 0.7 ** -1 * sqrt(0.3 * (1.0 + zgrid) ** 3 + 0.7)
```

Comment:
I was running this branch and found a couple of typos. Below is a suggested change to resolve the typos.

```suggestion
P20_rhoHI *= cosmo.h ** -1 * np.sqrt(cosmo.Om0 * (1.0 + zgrid) ** 3 + cosmo.Ode0)
P20_rhoHI /= 0.7 ** -1 * np.sqrt(0.3 * (1.0 + zgrid) ** 3 + 0.7)
P20_rhoHI_lo *= cosmo.h ** -1 * np.sqrt(cosmo.Om0 * (1.0 + zgrid) ** 3 + cosmo.Ode0)
P20_rhoHI_lo /= 0.7 ** -1 * np.sqrt(0.3 * (1.0 + zgrid) ** 3 + 0.7)
P20_rhoHI_hi *= cosmo.h ** -1 * np.sqrt(cosmo.Om0 * (1.0 + zgrid) ** 3 + cosmo.Ode0)
P20_rhoHI_hi /= 0.7 ** -1 * np.sqrt(0.3 * (1.0 + zgrid) ** 3 + 0.7)
```